src/components/Wizard/Wizard.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+
+export type WizardContextType = {
+  handleNext: () => void;
+  handleBack: () => void;
+  handleReset: () => void;
+  increaseStepBy: (increaseBy: number) => void;
+  decreaseStepBy: (decreaseBy: number) => void;
+  isFirstStep: boolean;
+  isLastStep: boolean;
+  currentStep: number;
+};
+
+const WizardContext = React.createContext<WizardContextType>(undefined);
+
+export const useWizardContext = () => React.useContext(WizardContext);
+
+export const Wizard: React.FC = ({ children }) => {
```

Comment:
Don't we have a PF Wizard component? Can it be used for our use case?

Reply:
There is a PF Wizard component but it has a sidebar that shows information related to step which is not needed for our use case.